app/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiLabOrderConverter.java

```diff
@@ -69,6 +55,10 @@ private MessageHeader createMessageHeader() {
         var messageHeader = new MessageHeader();
 
         messageHeader.setId(UUID.randomUUID().toString());
+        List<Coding> codingList = new ArrayList<>();
+        codingList.add(
+                0, new Coding("http://terminology.hl7.org/CodeSystem/v2-0103", "P", "Production"));
+        messageHeader.setMeta(new Meta().setTag(codingList));
```

Comment:
You don't need to be so verbose.  You also don't need to specify `0`, in `codingList.add` if you just want to append at the end of the `ArrayList`.

```suggestion
        messageHeader.setMeta(new Meta().addTag(new Coding("http://terminology.hl7.org/CodeSystem/v2-0103", "P", "Production")));
```